modules/FRDRAPIClient.py

```diff
@@ -0,0 +1,134 @@
+from util.util import Util
+from globus_sdk import RefreshTokenAuthorizer, NativeAppAuthClient
+from globus_sdk.base import BaseClient
```

Comment:
There is an updated Globus SDK Python client (v3) that changes some imports like this one.  See:
https://globus-sdk-python.readthedocs.io/en/stable/upgrading.html#upgrading

Reply:
Updated the version of globus_sdk to 3.1.0.